File: Soal-6/PRAK106-2410817310014-INDRA_SURYADILAGA.c
#include <stdio.h>

int main() {
    int a = 5;
    int b = 6;
    int c = 7;
    
    printf("Variabel a bernilai %d\n",a);
    printf("Variabel b bernilai %d\n",b);
    printf("Variabel c bernilai %d\n",c);
    
    printf("Apakah a sama dengan b? jawabannya adalah %d\n", a == b);
    printf("Apakah b lebih besar dari c? jawabannya adalah %d\n", b > c);
    printf("Apakah a tidak sama dengan b? jawabannya adalah %d\n", a != b);
    return 0;
}

File: Soal-6/PRAK106-2410817310014-INDRA_SURYADILAGA.py
a=5
b=6
c=7

charvar = ['a', 'b', 'c']
datavar = [a, b, c]

for char, data in zip(charvar, datavar):
    print(f'Variabel {char} bernilai {data}')

print(f'apakah a sama dengan b? jawabanya adalah {int(a==b)}')
print(f'apakah b lebih besar c? jawabanya adalah {int(a>b)}')
print(f'apakah a tidak sama dengan b? jawabanya adalah {int(a!=b)}')